sat/cli/bootsys/platform.py

```diff
@@ -480,6 +506,8 @@ def do_kubelet_start(ncn_groups):
     """
     do_service_action_on_hosts(ncn_groups['kubernetes'], 'kubelet',
                                target_state='active', target_enabled='enabled')
+    kube_api_waiter = KubernetesAPIWaiter(timeout=300)  # Adjust timeout as needed
+    kube_api_waiter.wait_for_completion()
```

Comment:
This needs to check the result of waiting on the `KubernetesAPIWaiter` and if it's unsuccessful, we should stop. Otherwise, as seen in your testing output, it goes ahead and tries to query Kubernetes for cronjobs, which will fail if the Kubernetes API is not available.